Dockerfile.frontend-container

```diff
@@ -1,15 +1,19 @@
-FROM node:18
+
+FROM node:16-alpine
+
 WORKDIR /usr/src/app
-COPY tornjak-frontend ./
-RUN npm install
+
+COPY tornjak-frontend .
+
+RUN npm install --omit=dev
+
+RUN npm prune --production
 
 # set env variables
 ENV REACT_APP_API_SERVER_URI $REACT_APP_API_SERVER_URI
 ENV REACT_APP_AUTH_SERVER_URI $REACT_APP_AUTH_SERVER_URI
-ARG NODE_OPTIONS=--openssl-legacy-provider
 ENV NODE_OPTIONS $NODE_OPTIONS
-
+ARG NODE_OPTIONS=--openssl-legacy-provider
 EXPOSE 3000
```

Comment:
We should parametrize the port as Env variable, so we can easily change it. It's ok for this PR, but we should open an issue for the future.

Reply:
Opened issue https://github.com/spiffe/tornjak/issues/153